zenith_utils/src/pq_proto.rs

```diff
@@ -798,3 +813,156 @@ impl<'a> BeMessage<'a> {
         Ok(())
     }
 }
+
+// Zenith extension of postgres replication protocol
+// See ZENITH_STATUS_UPDATE_TAG_BYTE
+#[derive(Debug, Clone, Copy, PartialEq, Serialize, Deserialize)]
+pub struct ZenithFeedback {
+    // Last known size of the timeline. Used to enforce timeline size limit.
+    pub current_instance_size: u64,
+    // Parts of StandbyStatusUpdate we resend to compute via safekeeper
+    pub ps_writelsn: u64,
+    pub ps_applylsn: u64,
+    pub ps_flushlsn: u64,
+    pub ps_replytime: SystemTime,
+}
+pub const ZENITH_FEEDBACK_FIELDS_NUMBER: u8 = 5;
```

Comment:
Lets and some note here. Something like: 
```
NOTE: Do not forget to increment this number when adding new fields to ZenithFeedback. Do not remove previously available fields because this might be backwards incompatible.
```